### ISSUE TITLE
The pytest_lazyfixture is not compatible with pytest 8

### ISSUE THREAD

**Issue by FrostyX** [elParaguayo/qtile-extras#388]
We use the `pytest_lazyfixture` here:

```bash
$ grep -r lazy_fixture
test/widget/test_currentlayouticon.py:from pytest_lazyfixture import lazy_fixture
test/widget/test_currentlayouticon.py:@pytest.mark.parametrize("currentlayout_manager", [lazy_fixture("temp_icons")], indirect=True)
```

It is not compatible with pytest 8, we should probably use https://github.com/dev-petrov/pytest-lazy-fixtures instead.

**Comment by elParaguayo** [comment]
Yes - so we currently force `pytest < 8.0.0`. If that link now works, do you want to submit a PR with this?

**Comment by FrostyX** [comment]
I can submit a PR. So far, I am trying to get this into the official Fedora repositories, so I am filing all the upstream issues that I need. But after that, I can try to solve them :-)

**Comment by dannycolin** [comment]
Someone found a clever solution to workaround this problem https://src.fedoraproject.org/rpms/qtile-extras/pull-request/6#request_diff
